scripts/addons/io_scene_gltf2/gltf2_generate.py

```diff
@@ -939,15 +939,15 @@ def generate_lights_cmn(operator,
     #
     #
 
-    filtered_lights_cmn = export_settings['filtered_lights_cmn']
+    filtered_lights = export_settings['filtered_lights']
 
-    for blender_light in filtered_lights_cmn:
+    for blender_light in filtered_lights:
 
         #
         # Property: light
         #
 
-        light = {'profile': 'CMN'}
+        light = {}
```

Comment:
Should the light's name be in here somewhere, or is that added through the node?

Reply:
Light name is exported. See line 977.